src/cluster_legacy.c

```diff
@@ -2748,6 +2764,60 @@ static clusterNode *getNodeFromLinkAndMsg(clusterLink *link, clusterMsg *hdr) {
     return sender;
 }
 
+static clusterMsgDataPublishMessage *getInitialBulkData(clusterMsgDataPublishMulti *msg) {
```

Comment:
These function names "bulk data" are too generic. They are specifically for the light publish multi, so name them something like that.

    getInitialPublishMessage, getNextCursorPublishMessage

or maybe

    getPublishMessageFirst, getPublishMessageNext